packages/api/test/e2e/api-derive/promise.spec.ts

```diff
@@ -7,13 +7,12 @@ import testingPairs from '@polkadot/keyring/testingPairs';
 import { WsProvider } from '@polkadot/rpc-provider';
 import { RewardDestination } from '@polkadot/types';
 
-import { HeaderExtended } from '../../src/type';
-import { DerivedBalances, DerivedFees, DerivedSessionInfo, DerivedStaking } from '../../src/types';
-import { SubmittableResult } from '../../../api/src';
+import { HeaderExtended } from '../../../../api-derive/src/type';
+import { DerivedBalances, DerivedFees, DerivedSessionInfo, DerivedStaking } from '../../../../api-derive/src/types';
```

Comment:
I think here and above we can just use `@polkadot/api-derive`;

Reply:
I got eslint errors for that, but that might be a confused VSCode after the the switch from tslint. I'll look into it.

Reply:
changed to `@polkadot` import